### PR TITLE
2.2

### PR DESCRIPTION
**Pull request by barcharcraz**

### CODE REVIEW

**Comment by sarum9in** [comment]
Doesn't build, could you please take a look what's wrong?

$ mesonwrap review tinyply --pull_request=8
Inspecting project tinyply, branch 2.2.
Repo name valid: ☑
Branch name valid: ☑
Target branch is not master: ☑
Has readme.txt: ☑
Has LICENSE.build: ☑
Has upstream.wrap: ☑
Has toplevel meson.build: ☑
Repo contains only buildsystem files: ☑
upstream.wrap has directory: ☑
upstream.wrap directory has no path separators: ☑
upstream.wrap has source_url: ☑
upstream.wrap has source_filename: ☑
upstream.wrap source_filename has no path separators: ☑
upstream.wrap has source_hash: ☑
Download url works: ☑
Hash matches: ☑
upstream.wrap directory 'tinyply-2.2' exists: ☑
Patch merges with source: ☑
```The Meson build system
Version: 0.46.0.dev1
Source dir: /tmp/tmpl9ncsuc9/src/tinyply-2.2
Build dir: /tmp/tmpl9ncsuc9/bin
Build type: native build
Project name: tinyply
Native C++ compiler: c++ (gcc 8.1.1 "c++ (GCC) 8.1.1 20180531")
Build machine cpu family: x86_64
Build machine cpu: x86_64
Build targets in project: 1
Found ninja-1.8.2 at /usr/bin/ninja
meson setup: ☑
ninja: Entering directory `/tmp/tmpl9ncsuc9/bin'
[1/3] Compiling C++ object 'tinyply@sha/source_tinyply.cpp.o'.
FAILED: tinyply@sha/source_tinyply.cpp.o 
c++ -Itinyply@sha -I. -I../src/tinyply-2.2 -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -std=c++11 -O0 -g -fPIC  -MD -MQ 'tinyply@sha/source_tinyply.cpp.o' -MF 'tinyply@sha/source_tinyply.cpp.o.d' -o 'tinyply@sha/source_tinyply.cpp.o' -c ../src/tinyply-2.2/source/tinyply.cpp
../src/tinyply-2.2/source/tinyply.cpp:5:10: fatal error: tinyply.h: No such file or directory
 #include <tinyply.h>
          ^~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.
```
ninja test: ☒

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,18 @@
+project('tinyply', 'cpp',
+    version: '2.2',
+    default_options: ['cpp_std=c++11'],
+    meson_version: '>=0.45.0')
+pkg = import('pkgconfig')
+
+i = include_directories('source')
+tinyply_lib = library('tinyply', 'source/tinyply.cpp',
+		      include_directories: i,
```
Comment (sarum9in) [inline meson.build]: Please use consistent indentation with spaces.

**Review by sarum9in — CHANGES_REQUESTED** [review]
Good overall, minor fix requested.

**Comment by barcharcraz** [comment]
I think I’ve resolved the issues

**Pull request merged**